File: C Programming/Assignments/Assignment 3 (For loop )/Q4.c
void main() {
    int a=7;
    int isPrime=1;
    for(int i=2;i<a;i++){
        if(a%i==0){
            isPrime=0;
            break;
        }
    }if(isPrime==1){
        printf("Prime");
}else{
        printf("Not Prime");
    }
}

File: Java/C TO JAVA/Assignment 3/Q4.java
class Q4{
public static void main(String args[]) {
    int a = 7;
    int i = 2;
    int isPrime = 1;
    while (i < a) {
        if (a % i == 0) {
            isPrime = 0;
            break;
        }
        i++;
    }
    if (isPrime == 1) {
        System.out.printf("Prime");
    } else {
        System.out.printf("Not Prime");
    }

}

}